### DISCUSSION TITLE
C++ plugin not found

### DISCUSSION THREAD

**Discussion by Hamza0256** [apache/netbeans#7604] [Q&A] [answered]
Hello,
After updating the Netbeans IDE , now there is no C++ plugin in Netbeans tools.

**Comment by YannLeCorse** [comment]
Not sure to understand your issue... could you share more details ? 

C++ seems installed and disabled by default on my fresh NB 22 install.
![Screenshot 2024-07-24 at 15 35 03](https://github.com/user-attachments/assets/eb9a3877-1bac-465f-b4fb-5621fa6b373a)

**Reply by Hamza0256** [reply]
Yes, I have the same plugin, but it only offers lightweight C++ support and isn't functioning properly. I can't open my project using this plugin. Please see the attached screenshot, which shows the tasks that can be performed with the correct plugin. However, I am currently unable to open my existing project in the newly updated IDE.
![Screenshot from 2024-07-24 09-46-45](https://github.com/user-attachments/assets/8427a4a7-2b5f-46d9-ab7d-527a8750563b)

**Comment by Hamza0256** [comment]
I used to install the C++ plugin from the available plugins in NetBeans, but in the updated version, the C++ plugin is no longer available.

**Reply by neilcsmith-net** [reply, marked as answer]
This overlaps with the discussion in #6979 The short answer is that this is no longer available.
